server/src/main/java/com/example/tyfserver/member/service/MemberService.java

```diff
@@ -113,6 +114,25 @@ public AccountInfoResponse accountInfo(LoginMember loginMember) {
         return AccountInfoResponse.of(member.getAccount());
     }
 
+    public void exchange(Long id) {
+        Member member = findMember(id);
+        Long exchangeablePoint = donationRepository.exchangeablePoint(id);
+        validateExchangeable(member, exchangeablePoint);
+
+        Exchange exchange =
+                new Exchange(exchangeablePoint, member.getAccount().getAccountNumber(), member.getNickname(), member.getPageName());
+        exchangeRepository.save(exchange);
+    }
+
+    private void validateExchangeable(Member member, Long exchangeablePoint) {
+        if (exchangeRepository.existsByPageName(member.getPageName())) {
+            throw new AlreadyRequestExchangeException();
+        }
+        if (exchangeablePoint <= 10000) {
```

Comment:
```suggestion
        if (exchangeablePoint < 10000) {
```
`정산 요청 - 이미 정산 요청을 한 상태인 경우`인수테스트가 깨지는데 10000원 미만으로 고쳐야 할듯!
그리고 예외 메시지도 `10000원 이하` -> `10000원 미만`으로 하고